service/assessment/assessment.go

```diff
@@ -242,6 +255,11 @@ func (svc *Service) AssessEvidence(_ context.Context, req *assessment.AssessEvid
 		return nil, status.Errorf(codes.InvalidArgument, "%v", err)
 	}
 
+	// Check if cloud_service_id in the service is within allowed or one can access *all* the cloud services
+	if !svc.authz.CheckAccess(ctx, service.AccessUpdate, req) {
```

Comment:
Is the context forwarded properly when called in `AssessEvidences` (`_, err = svc.AssessEvidence(context.Background(), assessEvidencesReq)`)?

Reply:
It was not :) good catch. I will forward `stream.Context()` to the call here.